demo.scenario

```diff
@@ -26,16 +26,16 @@ $ nix-shell -p nodejs
 --
 
 $ # Typing "nix-shell -p ..." each time can be tedious. We can do better.
-$ # We can write everything down in default.nix
+$ # We can write everything down in `default.nix`
```

Comment:
This is not really a markdown document. But something that would get "typed" into a terminal in asciinema video. If I would be typing this in a terminal I wouldn't use back-ticks. Can you remove the back-ticks?
```suggestion
$ # We can write everything down in default.nix
```